packages/manager/modules/web-paas/src/components/additional-option/template.html

```diff
@@ -0,0 +1,39 @@
+<div class="oui-tile d-flex m-2">
+    <h4 class="oui-tile__title ng-binding">
+        <span
+            data-translate="{{ 'web_paas_offer_additional_option_title_' + $ctrl.option.family}}"
+        ></span>
+    </h4>
+    <div class="text-right ml-auto mr-3">
+        <h4 class="m-0"><span data-ng-bind="$ctrl.getPrice()"></span></h4>
+        <span
+            data-translate="{{ 'web_paas_offer_additional_option_price_desc_' + $ctrl.option.family}}"
+        ></span>
+    </div>
+    <div class="oui-tile__body">
+        <oui-numeric
+            name="numeric"
+            model="$ctrl.option.quantity"
+            min="0"
+            disabled="$ctrl.disabled"
+            max="$ctrl.plan.getMaxLicenses()"
+            data-ng-if="$ctrl.option.family === 'user_license'"
```

Comment:
Could this be extracted to functions and with constants ? Not the best way to maintain value if it's in the HTML